src/components/memo-editor/LexicalMarkdownEditor/LexicalMarkdownEditor.toolbar.css

```
@@ -15,6 +15,75 @@
+}
+
+.scrollbar-hide::-webkit-scrollbar {
+  display: none;  /* Chrome, Safari and Opera */
+}
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  min-width: 36px;
+  min-height: 36px;
+  flex: 1;
```

Biome (1.9.4)

[error] Unexpected shorthand property flex after flex-shrink



(lint/suspicious/noShorthandPropertyOverrides)